docs/overview.md

```diff
@@ -0,0 +1,56 @@
+# qsim and qsimh
+
+qsim and qsimh are a collection of C++ libraries for quantum circuit
+simulation. These libraries provide powerful, low-cost tools for
+researchers to test quantum algorithms before running on quantum hardware.
+
+qsim makes use of AVX/FMA vector operations, OpenMP multithreading, and
+gate fusion [[1]](https://arxiv.org/abs/1601.07195)
+[[2]](https://arxiv.org/abs/1704.01127)
+to accelerate simuations. This performance is best demonstrated by the use
+of qsim in cross-entropy benchmarks here:
+[[3]](https://www.nature.com/articles/s41586-019-1666-5).
+
+Integration with [Cirq](https://github.com/quantumlib/Cirq) makes getting 
+started with qsim easy! Check out the
+[install guide](https://github.com/quantumlib/qsim/blob/master/docs/install_qsimcirq.md)
+or try the runnable
+[notebook tutorial](https://github.com/quantumlib/qsim/blob/master/docs/tutorials/qsimcirq.ipynb).
+
+## Design
+
+This repository includes two top-level libraries for simulation:
+
+-   **qsim** is a Schrödinger state-vector simulator designed to run on a
+    single machine. It produces the full state vector as output, allowing
```

Comment:
I would probably add "... output, for instance, allowing ..." or something like that as the full state vector allows to do other things like calculating observables. You don't really have to add that. What you wrote is also good.